### PR TITLE
docs: add arm template settings and arch

### PR DESCRIPTION
**Pull request by noly**
In this PR I added information about how the template and deploy to azure works

### CODE REVIEW

File: EventHubForwarder/README.md [hunk 1]
```diff
@@ -1,6 +1,48 @@
-This function collects logs (e.g. Activity and Diagnostic logs) from Azure Event Hubs and forwards them to New Relic.
+An Azure Resource Manager template to export Azure Platform logs to New Relic.
 
-## Create the Azure Function App
+## How Does It Work?
+This integration creates and configures the Azure resources necessary to efficiently forwards logs from an Azure Event Hub to New Relic. 
+It relies on events managed by Azure Event Hub, this Event Hub subsequently batches and triggers an Azure Function to handle the transport to New Relic.
```
Comment (jodstrcil) [inline EventHubForwarder/README.md:5]: this Event Hub -> Event Hub

File: EventHubForwarder/README.md [hunk 2]
```diff
@@ -1,6 +1,48 @@
...
+
+Currently, this integration allows you to create resources to targets Azure Activity logs. If you have other log events that you would like to see exported using Event hub trigger, [tell us about your use case](https://github.com/newrelic/newrelic-azure-functions/issues).
```
Comment (jodstrcil) [inline EventHubForwarder/README.md:7]: exported or send?

File: EventHubForwarder/README.md [hunk 3]
```diff
@@ -1,6 +1,48 @@
...
+
+## Installation
+This integration requires both a New Relic and Azure account.
+
+### Install Using Azure Portal
+Retrieve your [New Relic Insights Insert Key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/new-relic-api-keys/#insights-insert-key).
```
Comment (jodstrcil) [inline EventHubForwarder/README.md:13]: License keys?

**Review by jodstrcil — COMMENTED** [review]
Nice 🎉

File: EventHubForwarder/README.md (hunk 2 above)
Comment (noly) [reply EventHubForwarder/README.md:7]: 🚢 shipped

File: EventHubForwarder/README.md (hunk 1 above)
Comment (noly) [reply EventHubForwarder/README.md:5]: Thanks

File: EventHubForwarder/README.md (hunk 3 above)
Comment (noly) [reply EventHubForwarder/README.md]: I deleted all that part ✂️ 🔪

File: EventHubForwarder/README.md [hunk 4]
```diff
@@ -1,6 +1,48 @@
...
+| Enable Security Azure Activity Logs | no | `false` | Contains the record of any alerts generated by Azure Security Center. More information about Security category in [azure official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema#security-category). |
+| Enable Service Health Azure Activity Logs | no | `false` | Contains the record of any service health incidents that have occurred in Azure. More information about Recommendation category in [azure official documentation](. More information about Recommendation category in [azure official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema#recommendation-category). |
+
+Alternatively you can configure this template that will automatically create the resource, deployed it and configure Activity Logs to forward them to New Relic.
```
Comment (jsubirat) [inline EventHubForwarder/README.md:40]:
```suggestion
Alternatively you can configure this template that will automatically create the resources, deploy them and configure Activity Logs to forward them to New Relic.
```

**Review by jsubirat — APPROVED** [review]

**Pull request merged**